examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureMultiplicationExample.tsx

```diff
@@ -1,9 +1,16 @@
 // (C) 2007-2019 GoodData Corporation
 import React from "react";
 import { PivotTable } from "@gooddata/sdk-ui-pivot";
+import { newArithmeticMeasure, measureLocalId } from "@gooddata/sdk-model";
 import { Ldm, LdmExt } from "../../ldm";
 
-const measures = [LdmExt.NrRestaurants, LdmExt.averageRestaurantDailyCosts, LdmExt.arithmeticMeasure2];
+const averageStateDailyCosts = newArithmeticMeasure(
+    [measureLocalId(LdmExt.NrRestaurants), measureLocalId(LdmExt.averageRestaurantDailyCosts)],
```

Comment:
you should be able to omit measureLocalId, just use the measures themselves

Reply:
Good catch, removed